experimental/regression_suite/tests/pregenerated/test_llama2.py

```diff
@@ -19,7 +19,7 @@
 ]
 
 llama2_7b_f16qi4_source = fetch_source_fixture(
-    "https://storage.googleapis.com/shark_tank/stella/llama2_7b_int4_stripped.mlir",
```

Comment:
Do you want to keep a known URL in `https://storage.googleapis.com/shark_tank/iree-tests/<SHA>` or such ?

Reply:
We ultimately want all of these coming from common places like that which we update with a generation script. However, as long as we're in this phase with this one of it being a bit hand-massaged, I'm ok with Dan taking responsibility to keep it updated.